### ISSUE TITLE
Elevated plasma glutarylcarnitine, C5-DC

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5962]
**Preferred term label:** Elevated plasma glutarylcarnitine, C5-DC

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated Glutarylcarnitine concentration in the blood., PMID: 15617193

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
